patroni/postgresql.py

```diff
@@ -164,9 +167,42 @@ def query(self, sql, *params):
     def data_directory_empty(self):
         return not os.path.exists(self.data_dir) or os.listdir(self.data_dir) == []
 
+    @staticmethod
+    def initdb_allowed_option(name):
+        allowed_options = set(['auth', 'auth-host', 'auth-local', 'encoding', 'data-checksums',
+                               'locale', 'lc-collate', 'lc-ctype', 'lc-messages', 'lc-monetary',
+                               'lc-numeric', 'lc-time', 'text-search-config', 'xlogdir', 'debug', 'noclean'])
```

Comment:
I know initdb options are changing infrequently, but does it really make sense to hard-code the known list here?  It will fail anyway if some option or passed value is invalid.